### PR TITLE
Add node start delay in seconds

### PR DESCRIPTION
**Pull request by piotrpekala7**

### CODE REVIEW

File: gns3server/controller/project.py [hunk 1]
```diff
@@ -1064,17 +1064,18 @@ def dump(self):
...
 
     @open_required
-    async def start_all(self):
+    async def start_all(self, delay = 0):
         """
         Start all nodes
         """
         pool = Pool(concurrency=3)
         for node in self.nodes.values():
+            await asyncio.sleep(delay)
```
Comment (piotrpekala7) [inline gns3server/controller/project.py:1073]: I see that we are using asyncio.sleep in pool.py#45 but I'm not sure if it's good practise. It it working in that case?

Comment (grossmj) [reply gns3server/controller/project.py:1073]: I don't think this works. Probably the delay should be implemented in node.start()

Comment (ziajka) [reply gns3server/controller/project.py:1073]: Yes, here https://github.com/GNS3/gns3-server/blob/master/gns3server/compute/qemu/qemu_vm.py#L942 would be good start

**Comment by grossmj** [comment]
Implementation will be different on version 3.0

**Pull request closed without merging**